### PR TITLE
Add basic airgap documentation

### PR DESCRIPTION
**Pull request by suligap**

### CODE REVIEW

**Comment by doismellburning** [comment]
In terms of general readability and clarity it looks good, thank you!

I don't have enough familiarity to review for technical accuracy off the top of my head without actually running through it all myself and I'm not sure when I'd be able to do that - hopefully @bloodearnest can provide sufficient technical/functional input though!

File: en/airgap.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Example of importing a `jq.tar.gz` snap bundle on the air-gapped proxy host:
+
+```bash
+sudo snap-store-proxy push-snap jq.tar.gz
+```
+
+The `jq` snap is now available for installation from this air-gapped Snap Store
+Proxy.
+
```
Comment (bloodearnest) [inline en/airgap.md:126]: Maybe add a way to test this from a machine inside the airgap? e.g. snap info jq

File: en/airgap.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+## Client Device Configuration
+
+Client devices only ever connect to the offline proxy. They do this without
+sending any device authentication/authorization information to the proxy. A
+client device that has contacted the official Snap Store before, and has
+obtained a
+[serial assertion](https://docs.ubuntu.com/core/en/reference/assertions/serial),
+will not be able to use the air-gapped proxy for example.
```
Comment (bloodearnest) [inline en/airgap.md:134]: ..., as the airgap proxy currently is unable to authenticate it offline. 
?

**Comment by suligap** [comment]
Thank you for the reviews!

**Pull request merged**